posts/2017-10-10-serverless-conf-2017-nyc-recap.md

```diff
@@ -0,0 +1,155 @@
+
+Mark gives it a **B**. (Better, he notes, than containers)
+
+# 10 tips for running a serverless business... number #6 will blow your mind!
+by [Sam Kroonenburg](https://twitter.com/samkroon)
+
+Sam started coding a learning platform 4 years ago. You might know it now as this little company called [A Cloud Guru](acloud.guru). He knew he'd need to include video lessons, a quiz engine, an online store and sign up / log in, while having something that scaled effortlessly and had low operational overhead.
+
+Tl;dr: he cared about the fastest, cheapest way to build a company and get his MVP out there. So he went serverless (check out his EC2 for proof):
+
+<img width="600" src="https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/slsconf+2017+recap/ec2.jpg">
+
+His learning platform started as a serverless monolith, and Sam advocates that choice. It's a great way to launch fast, and as long as you're starting serverless, the transition to microservices later is pretty straightforward.
+
+For all those serverless entrepreneurs out there, Sam has some guiding principles and advice:
+
+- It's a myth that serverless means you don't need ops. Of course you need ops, but responsibility *does* shift to the dev team
+- When you need to fill engineering positions, don’t look for serverless developers. They don’t exist yet. Instead, filter for a developer who cares about smooth running code in production.
+- Encourage & reassure job candidates that it's okay if they don't know your stack; you're ready to teach, and it will be exciting for them. Otherwise, they'll be too intimidated to apply at all.
+- Join the community. That's how Sam learned about Algolia back in the day, and began contributing to the Serverless Framework.
+- Expect to pioneer. Serverless teams build more tools, all the time.
+- It’s ok to build a serverless monolith. And when you decide to migrate to a microservices architecture, you can do it without thinking about infrastructure.
+- Automation is not optional. You can’t deploy all these pieces manually once you have several.
+- Test all the things. You can never fully emulate operating environments for development purposes.
+
+# Shipping Containers As Functions
+
+by [Amiram Shachar]()
+
+Remember how, with every technological invention, the word on the street was that the 'old' would completely go away? Computers would kill paper, Microsoft would kill IBM, etc. In reality, this process takes way, way longer than we think.
+
+So what does the future of serverless look like? Here's what Amiram thinks:
+
+<img width="600" src="https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/slsconf+2017+recap/futuresls.jpg">
+
+As you can see, containers and VMs are still in that chart. He also makes a case for shipping containers as serverless functions. You could, say, use Docker image as function. You wouldn't have to zip anything or add dependencies; just put it in a docker file, package and ship.
```

Comment:
1) `use Docker image as function.` --> `use a Docker image as a function.`

2) Capitalize Docker in last sentence.